ui/components/ui/popover/index.scss

```diff
@@ -51,7 +51,7 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    z-index: 1050;
+    z-index: $modal-z-index;
   }
```

Comment:
Shows no change to Popover z-index and that it still displays as intended in production on the Terms of Use modal

<img width="1438" alt="Screenshot 2023-05-06 at 9 59 36 AM" src="https://user-images.githubusercontent.com/8112138/236588851-81a45cbd-7afc-4e11-a723-5f4a31862d5b.png">

<img width="1440" alt="Screenshot 2023-05-06 at 9 55 24 AM" src="https://user-images.githubusercontent.com/8112138/236588848-29121335-c34d-4db4-8dca-8bd167c1ca6c.png">